centreon/www/front_src/src/Resources/Filter/criteriasNewInterface/extendedFilter/FilterSearch/useDebounce.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useMemo, useRef } from 'react';
```

Comment:
There is already a useDebounce hook here: https://github.com/centreon/centreon/blob/develop/centreon/packages/ui/src/utils/useDebounce.ts